cityofaustin/atd-vz-data
# Add an address search feature to the VZV map

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/11581

This PR updates the crash map with a geocoder search. It also updates the way the map ref is assigned (note in the code diff about this that points to some docs).

## Testing
**URL to test:** 
https://deploy-preview-1217--atd-vzv-staging.netlify.app/viewer/map

**Steps to test:**
1. Test the address search by entering a street, place, etc within the Austin area and select it from the dropdown
2. You should see the map fly to the location that you selected
3. Try searching for a place outside of the Austin area (Dallas, San Antonio, or wherever) and you'll see that results do not populate for locations outside of the navigable map area
4. To test the updates to features that could have been affected by the `ref` code change:
   - Test mobile view and you'll see that, when you click on a crash dot on the map, the popup info box opens towards the viewable space even if it is near the edge of the map (towards the horizontal center - the popup can still open off-screen vertically)
   - Toggle fatal and serious injury crash filters
   - Make sure the loading compass spinner shows when data is loading on the map

---
#### Ship list
